Automattic/wp-calypso
# Update atomic eligibility copy for unlaunched and private sites

#### Changes proposed in this Pull Request

Part of the #38141, this PR only includes wording changes for telling the user their site must be launched. Just to keep the wording changes easier to review. The bigger style changes are in #38277

* Change message shown for _unlaunched_ site with `SITE_PRIVATE` hold.
* Change message shown for _launched_ site with `SITE_PRIVATE` hold.

*Message for unlaunched site*
<img width="653" alt="Screenshot 2019-12-11 at 10 14 40 AM" src="https://user-images.githubusercontent.com/1500769/70569949-d0af9600-1bff-11ea-8748-1472c597a6b4.png">

*Message for launched private site*
<img width="695" alt="Screenshot 2019-12-11 at 10 26 24 AM" src="https://user-images.githubusercontent.com/1500769/70570480-da85c900-1c00-11ea-9cbe-3d81fbaf899c.png">
The help link goes to https://en.support.wordpress.com/settings/privacy-settings/, which is currently what production does. The button will probably be removed as part of #38141 since proceeding will change the setting for the user.

#### Testing instructions



* Choose a simple site to test that's unlaunched
  * Go to plugins, click upload plugin
  * Should see message that site must be launched
* Choose a simple simple site that's launched but private
  * Go to plugins, click upload plugin
  * Should see message that site must be made public
